src/commands/generate/models.ts

```diff
@@ -142,8 +142,10 @@ export default class Models extends Command {
     const { tsModelType, tsEnumType, tsIncludeComments, tsModuleSystem, tsExportType, tsJsonBinPack, namespace, csharpAutoImplement, csharpArrayType, csharpNewtonsoft, csharpHashcode, csharpEqual, csharpSystemJson, packageName, output } = flags;
     const { language, file } = args;
     const inputFile = (await load(file)) || (await load());
-    const { document, status } = await parse(this, inputFile, flags);
+    const { document, diagnostics ,status } = await parse(this, inputFile, flags);
     if (!document || status === 'invalid') {
+      const severityErrors = diagnostics.filter((obj) => obj.severity === 0);
+      this.log(`Input is an empty file or not a correct AsyncAPI document so it cannot be processed.${formatOutput(severityErrors,'stylish','error')}`);
```

Comment:
@chinma-yyy looking at your screenshot I have a question why does it say that input is an empty file because it is not, you are showing warning on whats wrong with the file, so it is not an empty file. 

What I am trying to say is, you have two situations on error 
1. path file is wrong and the spec does not exist 
2. path file is right but there is some issue with the file. 

I think you should change the error message.

Reply:
Yup, thought that the `document` object would be undefined if the file is empty that's why we have made the `!document` condition there. Anyways I have made the changes